README.rst

```diff
@@ -88,11 +88,22 @@ Installation
 
        easy_install aiosparql
 
+
 Requirements
 ^^^^^^^^^^^^
 
  *  Python >= 3.5
```

Comment:
(The version requirement should match the tests of the CI)

Reply:
Agreed